deployment/build.portable

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+#= Portable .NET DLL .tar.gz
```

Comment:
```suggestion
```

Reply:
This needs to exist. It's how the build script lists its platforms in a nice way.